### DISCUSSION TITLE
Great!  Any chance to make it work with Groupy?

### DISCUSSION THREAD

**Discussion by duerra** [manutalcual/winredock#50] [General]
So thankful for this project.  The one issue I have is that I use Groupy to combine a number of my different windows together (primarily groups of browser windows and their associated tabs).  When this executes, it breaks apart all of my Groupy windows.  If possible, it would be perfect to have this maintain that singular window grouping!

**Comment by manutalcual** [comment]
Thanks for the feedback. Unfortunately, I don't have time to do that. Sorry. I guess it may be easier to ask Stardock to add my code to their app than to try to make both work seamlessly, although they will have to refactor my code a lot.
I'm sorry.

**Comment by duerra** [comment]
Thanks for the response!
